### PR TITLE
add link to build instructions

### PR DESCRIPTION
**Pull request by joostholslag**

### CODE REVIEW

**Comment by sebastian-iancu** [comment]
This (original) text was more informative about the purpose of the repo.
Your addition is good, but I think the right place is somewhere at https://openehr-nl.github.io/help.html.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # openEHR Nederland workspace
 
 This repository is used to generate website at [https://openehr-nl.github.io](https://openehr-nl.github.io).
+For instructions to build and run locally: https://github.com/openehr-nl/openehr-nl.github.io/blob/master/_build.md
```
Comment (joostholslag) [inline README.md]:
```suggestion
```

**Comment by joostholslag** [comment]
Agreed, how about this? @sebastian-iancu

**Comment by sebastian-iancu** [comment]
there is something strange in this PR, your submodule I think is updated to a wrong checkpoint (old commit ref)

**Comment by sebastian-iancu** [comment]
changes merged manually

**Pull request closed without merging**